src/main/java/frc/robot/utilities/EncoderConversions.java

```diff
@@ -31,6 +31,16 @@ public static double falconToRPM(double velocityCounts, double gearRatio) {
         return mechRPM;
     }
 
+    /**
+     * Default gear ratio of 1
+     *
+     * @param velocityCounts Falcon Velocity Counts
+     * @return RPM of Mechanism
+     */
+    public static double falconToRPM(double velocityCounts) {
+        return falconToRPM(velocityCounts, 1);
+    }
```

Comment:
no default functions here

Reply:
it is not my case, should I still change it?